girder/taskflow/server/models/taskflow.py

```diff
@@ -61,6 +61,7 @@ def append_to_log(self, taskflow, log):
             }
         }
 
+        send_log_notification('taskflow', taskflow, log)
         return self.update(query, update, multi=False)
 
```

Comment:
No, after the call to update. So the end of the function would look something like this:

```
result = self.update(query, update, multi=False)
send_notification('taskflow', 'status', taskflow)

return result
```